rewrite-java/src/main/java/org/openrewrite/java/internal/parser/TypeTable.java

```diff
@@ -348,11 +358,6 @@ public MethodVisitor visitMethod(int access, String name, String descriptor,
                     throw new UncheckedIOException(e);
                 }
             }
-
-            public ClassDefinition classDefinition(int access, String name, @Nullable String signature,
-                                                   String superclassName, String @Nullable [] superinterfaceSignatures) {
-                return new ClassDefinition(this, access, name, signature, superclassName, superinterfaceSignatures);
```

Comment:
Removed this indirection as we're merely calling a constructor. I don't recall this method being used elsewhere just yet.